### DISCUSSION TITLE
Dynamic table Add new row and input

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#13405] [CLI - PWA mode] [answered]
Hello Friends 

Am having issues with my dynamic table am not able to figure out the solution.  
I wanted to have 2 column 1 **Min** and the other **Max** , when ever I have a value in column **Max** it should add a new row in which the column **Min** should have the value of the max column plus 1 .

for example 
![image](https://user-images.githubusercontent.com/84211129/168000745-83bb4ee3-fd0d-4c3a-8818-0ad05a759afc.png)

and every time i delete a row it should take the value of the max column plus 1.

am not able to find i got really stuck 

Here is the code pen that i have created 
https://codepen.io/akashramtohul4/pen/LYQZKqK?editors=1010

i would be grateful if someone could help me on that 

Thanks

**Comment by metalsadman** [comment, marked as answer]
https://codepen.io/metalsadman/pen/GRQjZQY


> and every time i delete a row it should take the value of the max column plus 1.

I didn't quite get or understood this part (which should take the value?), so just edit/improve the codepen according to your requirements. goodluck

**Reply by akashramtohul4** [reply]
Hi Metal for helping me. let me make myself clear with the explanation.
![image](https://user-images.githubusercontent.com/84211129/168485136-3bbbd169-f749-4cfc-8a37-661073a1f527.png)

Every time i add a value in max field it should add it in the next row of min, it should not add a new row every time i insert a value only the next row should be affected.

And if i delete the row below it should take the value in the max above plus 1 
![image](https://user-images.githubusercontent.com/84211129/168486030-5b28978e-6c8d-42e9-942c-01ee89108b50.png)

can add some comments so that i can understand your thinking process which helps me to improve my  
Thanks for helping metal

**Reply by metalsadman** [reply]
updated the codepen https://codepen.io/metalsadman/pen/GRQjZQY

https://user-images.githubusercontent.com/31038838/168519454-1f9f368b-2286-4a2f-8038-483948f69721.mov

**Reply by akashramtohul4** [reply]
Thanks  Metal ,

You have really been a great help. I need you assistance on one more thing i have evolve the code by adding 2 more column Monthly and Yearly which are mandatory fields and i have save button . In short the save button remain disable when the Monthly or  Yearly field are empty  in any row.  But its not working for all rows only the first one is working 

Here is the Sample code :  https://codepen.io/akashramtohul4/pen/rNJYYgo?editors=1011

Can you please help me on that 

Thanks
